plugins/woocommerce/src/Internal/Admin/Suggestions/PaymentExtensionSuggestions.php

```diff
@@ -2867,6 +2869,39 @@ private function get_all_extensions_base_details(): array {
 					),
 				),
 			),
+			self::MONEI             => array(
+				'_type'       => self::TYPE_PSP,
+				'title'       => esc_html__( 'MONEI', 'woocommerce' ),
+				'description' => esc_html__( 'Accept Cards, Apple Pay, Google Pay, Bizum, PayPal, and many more payment methods in your store.', 'woocommerce' ),
+				'image'       => plugins_url( 'assets/images/onboarding/monei.svg', WC_PLUGIN_FILE ),
```

Comment:
No need for the image ATM.

Reply:
removed in [06d8d34](https://github.com/woocommerce/woocommerce/pull/55363/commits/06d8d34f6ee9f3d82ee81e3662fa6c2ff4d36a20) . thanks!